pkg/network/tracer/connection/kprobe/config.go

```diff
@@ -9,13 +9,30 @@ package kprobe
 
 import (
 	"fmt"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
 
 	"github.com/DataDog/datadog-agent/pkg/ebpf"
 	"github.com/DataDog/datadog-agent/pkg/network/config"
 	"github.com/DataDog/datadog-agent/pkg/network/ebpf/probes"
 	"github.com/DataDog/datadog-agent/pkg/util/kernel"
 )
 
+// After kernel 6.5.0, tcp_sendpage and udp_sendpage are removed.
+// We used to only check for kv < 6.5.0 here - however, OpenSUSE 15.6 backported
+// this change into 6.4.0 to pick up a CVE so the version number is not reliable.
+// Instead, we directly check if the function exists.
+func getHasSendPage(kv kernel.Version) bool {
```

Comment:
Just `hasSendPage` is more golang-y: https://google.github.io/styleguide/go/decisions#getters

Reply:
I changed the name slightly to avoid shadowing but lmk if there is a better way:
`hasSendPage := hasTCPSendPage(kv)`